ui/components/app/gas-details-item/gas-details-item-title/index.scss

```diff
@@ -6,4 +6,11 @@
     color: var(--color-text-muted);
     line-height: inherit;
   }
+
+  &__estimate-used {
+    font-weight: 400;
+    font-size: 12px;
```

Comment:
We should try to avoid using static CSS here in favour of the `Text` component https://metamask.github.io/metamask-storybook/?path=/docs/components-componentlibrary-text--docs